src/public/modules/forms/admin/controllers/list-forms.client.controller.js

```diff
@@ -40,21 +40,6 @@ function ListFormsController(
   // Redirect to signin if unable to get user
   vm.user = Auth.getUser() || $state.go('signin')
 
-  vm.openContactNumberModal = () => {
-    $uibModal
```

Comment:
when I said "changes and reverts" I was referring to stuff like this - why is this written in the frontend PR then deleted here?

Reply:
It caused a bug; see 5b2bb5d

Reply:
Oh wait that commit doesn't explain anything; basically if you logged in but got logged out since your session ended, you will be moved to the login page but the contact number modal will still open in the login page, since it is coded to only run when all angular digest cycles have resolved.

By moving it to the avatar dropdown state, it is
1. closer to its origin
2. that situation will not happen since it only opens if the latest user fetched from the server is valid